### PR TITLE
UI - support elastic search reindexing from UI

### PR DESCRIPTION
**Pull request by chirag-madlani**
### Describe your changes :

I worked on the closes: #5952

#
### Type of change :

- [x] Bug fix
- [x] Improvement
- [x] New feature
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation

#
### Frontend Preview (Screenshots) :
Screenshot 2022-09-29 at 11 33 04 AM

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#

### CODE REVIEW

File: openmetadata-ui/src/main/resources/ui/src/pages/ElasticSearchIndexPage/elastic-search-index-page.tsx [hunk 1]
```diff
@@ -0,0 +1,268 @@
...
+  const [batchLoading, setBatchLoading] = useState(false);
+  const [streamLoading, setStreamLoading] = useState(false);
+
+  const { permissions } = usePermissionProvider();
+
+  const createPermission = useMemo(
+    () => checkPermission(Operation.All, ResourceEntity.USER, permissions),
+    [permissions]
+  );
+
```
Comment (Sachin-chaurasiya) [inline openmetadata-ui/src/main/resources/ui/src/pages/ElasticSearchIndexPage/elastic-search-index-page.tsx:53]: Can we check if this is correct as here we are checking for User Resource?

File: openmetadata-ui/src/main/resources/ui/src/styles/position.less [hunk 2]
```diff
@@ -14,3 +14,11 @@
 .align-start {
   align-items: start;
 }
+
+.d-flex {
+  display: flex;
+}
+
+flex-1 {
```
Comment (Sachin-chaurasiya) [inline openmetadata-ui/src/main/resources/ui/src/styles/position.less:22]:
```suggestion
.flex-1 {
```

File: openmetadata-ui/src/main/resources/ui/src/utils/TableUtils.tsx [hunk 3]
```diff
@@ -355,3 +356,35 @@ export const getTestResultBadgeIcon = (status?: TestCaseStatus) => {
...
+
+    case Status.Activewitherror:
+      return 'Active with error';
+
+    case Status.Retry:
+      return 'Retry';
+
+    default:
+      return status || '';
+  }
```
Comment (Sachin-chaurasiya) [inline openmetadata-ui/src/main/resources/ui/src/utils/TableUtils.tsx:389]: Can we move this to EventPublisher Utils?

**Review by Sachin-chaurasiya — APPROVED** [review]

File: openmetadata-ui/src/main/resources/ui/src/styles/position.less [hunk 4]
```diff
@@ -14,3 +14,11 @@
 .align-start {
   align-items: start;
 }
+
+.d-flex {
+  display: flex;
+}
```
Comment (ShaileshParmar11) [inline openmetadata-ui/src/main/resources/ui/src/styles/position.less:20]: We already have flex class in app.less

File: openmetadata-ui/src/main/resources/ui/src/pages/ElasticSearchIndexPage/elastic-search-index-page.tsx [hunk 5]
```diff
@@ -0,0 +1,259 @@
...
+    fetchBatchReIndexedData();
+    fetchStreamReIndexedData();
+  };
+
+  useEffect(() => {
+    fetchData();
+  }, []);
+
+  return (
+    <div>
```
Comment (ShaileshParmar11) [inline openmetadata-ui/src/main/resources/ui/src/pages/ElasticSearchIndexPage/elastic-search-index-page.tsx:94]: we can avoid extra <div> wrapping

**Review by ShaileshParmar11 — APPROVED** [review]
LGTM! Left miner comments!

**Pull request merged**